File: problemas/iniciante/1541/1541.rb
def main
    loop do
      begin
        input = gets.chomp
        break if input.nil? || input.strip.empty?
  
        a, b, c = input.split.map(&:to_i)
        break if a == 0
  
        result = Math.sqrt(a * b * 100.0 / c).floor
        puts result
  
      rescue StandardError
        break
      end
    end
  end
  
  main

File: problemas/iniciante/1541/1541.py
import math

def main():
    while True:
        try:
            A, B, C = map(int, input().split(' '))

            if A == 0:
                break

            print(f"{math.floor(math.sqrt(A * B * 100 / C)):.0f}")
        
        except EOFError:
            break
        except ValueError:
            break

if __name__ == "__main__":
    main()
